flatfs_test.go

```diff
@@ -343,6 +343,8 @@ func testQuerySimple(dirFunc mkShardFunc, t *testing.T) {
 	temp, cleanup := tempdir(t)
 	defer cleanup()
 
+	ioutil.WriteFile(filepath.Join(temp, "README"), []byte("something"), 0666)
```

Comment:
might want to check that this doesnt error, just in case (if it errors and fails to create the file, then the test would pass regardless of the above fix)